### DISCUSSION TITLE
Will the pods be able to communicate without any mTLS issues when one pod runs with a Linkerd sidecar and the other doesn't?

### DISCUSSION THREAD

**Discussion by saikrishnakumarreddy** [linkerd/linkerd2#12481] [Q&A]
Will the pods be able to communicate without any mTLS issues when one pod runs with a Linkerd sidecar and the other doesn't?

**Comment by alpeb** [comment]
Yes, pods should still be able to connect to one another if either of them is not meshed, and those connections won't be wrapped in mTLS.
